docs/site/migration/auth/example.md

```diff
@@ -0,0 +1,570 @@
+This example is migrated from
+[loopback-example-access-control](https://github.com/strongloop/loopback-example-access-control),
+and uses the authentication and authorization system in LoopBack 4 to implement
+the access control.
+
+# App Scenario
```

Comment:
The headings should start with the second-level ones `#` -> `##` (so shift all of them by one)